test/test_gbcon.cc

```diff
@@ -66,7 +65,7 @@ void test_gbcon_work( Params& params, bool run )
     int64_t info_tst = lapack::gbcon( norm, n, kl, ku, &AB[0], ldab, &ipiv_tst[0], anorm, &rcond_tst );
     time = testsweeper::get_wtime() - time;
     if (info_tst != 0) {
-        fprintf( stderr, "lapack::gbcon returned error %lld\n", (lld) info_tst );
+        fprintf( stderr, "lapack::gbcon returned error %lld\n", llong( info_tst ) );
```

Comment:
Where is the `llong` defined?

Reply:
Found in include/lapack/utils.hh.

Reply:
It's in 2 places: for code inside the `lapack` namespace, it's in `include/lapack/utils.hh`. For the testers, which are outside the `lapack` namespace, it's also in `test/test.hh`. That avoids repeating it in every file, and having to remove it if the compiler complains that it is unused in a particular file.